docs/content/anonymization/command.md

```diff
@@ -0,0 +1,125 @@
+---
+outline:
+   - 2
+   - 3
+---
+
+# Anonymization command
+
+Considering your anonymization has been configured, you can
+now anonymize a backup file by running:
+
+```sh
+console db-tools:anonymize path/to/your/backup/to/anonymized
+```
+
+This command will successively:
+
+1. Backup your local database,
+2. Restore the given backup file,
+3. Anonymize the data from the given backup file,
+4. Backup the newly anonymized database, **by overwritting the given backup file**,
+5. Restore your database to its original state from the backup produced at step 1.
+
+::: warning
+The `db-tools:anonymize` command alone is not enough to ensure you follow GDPR best practices.
+It depends on:
+
+* How you correctly configured your anonymization (obviously),
+* Where you run this command: anonymizing a backup file means it contains
+  sensitive data, hence, following GDPR recommendations, this **backup file
+  should never transit on an unsecured environment**.
+
+Read the next section to learn more about a proper workflow.
+:::
+
+## A GDPR-friendly workflow
+
+Here is an example of workflow - that follows GDPR recommendations - to retrieve anonymized production
+data on your local environment.
+
+### Prerequisites
+
+* You have 2 secured environments : *production* and *another_env* (such as a preproduction)
+  and you can securely copy files from one to another,
+* You can stop your service on *another_env*,
+* Your anonymization is well configured, every sensitive data has been
+  mapped to an anonymizer to erased/hashed/randomized it.
+
+::: note
+Note that *another_env* could be any environment, not only preproduction, all it needs is the
+the Symfony Console to work and a database. It doesn't need to be a complete working env.
+:::
+
+### Workflow
```

Comment:
```suggestion
### Workflow

Let's call *another_env* the environment we have besides the production one.
```

Reply:
Or "Let's assume the environment we have besides the production one is called *another_env*."